packages/editor/src/components/post-actions/actions.js

```diff
@@ -39,6 +39,17 @@ import { CreateTemplatePartModalContents } from '../create-template-part-modal';
 const { PATTERN_TYPES, CreatePatternModalContents, useDuplicatePatternProps } =
 	unlock( patternsPrivateApis );
 
+// TODO: this should be shared with other components (page-pages).
```

Comment:
Consolidation can be done in a follow-up PR, but I wanted to gather early thoughts: do we create a registry for fields per postType? where do we extract the Field information for now? Actions are defined in the `editor` package while the `edit-site` package defines the DataViews.

Reply:
Yeah, I think right now the registry should be in the. `editor/src/dataviews/fields` folder like we have `editor/src/dataviews/actions` folder if we're talking about the default WP entities fields.

My thinking is that I'll probably move `editor/src/dataviews` to either a dedicated package `@wordpress/core-dataviews` or just move it into `@wordpress/core-data` because it's actually lower level than the editor and can be used outside the editor.